base_ir_action_group_access/controllers/main.py

```diff
@@ -11,6 +11,15 @@ class Action(Action):
     @http.route("/web/action/load", type="json", auth="user")
     def load(self, action_id, additional_context=None):
         action_access = False
+        try:
+            action_id = int(action_id)
+        except ValueError:
```

Comment:
@TimLai125 Can you add info regarding in which case this happens?

Reply:
Actually here I just follow the standard Odoo logic, the standard logic checks the action_id in this method, i.e. https://github.com/odoo/odoo/blob/12.0/addons/web/controllers/main.py#L1240-L1251

Reply:
Ok, according to this https://github.com/odoo/odoo/commit/71a5c53072341f784d1eb87e561f59823fbb1651, this is for the case action_id is an xml id.